src/components/Room.jsx

```diff
@@ -8,21 +8,22 @@ import {
   LEAVE_ROOM,
 } from '../sockets/SocketConnection';
 import { Nav } from './Nav';
-
-const POINTERS = 'pointers';
-const OBSERVERS = 'observers';
+import { Scale } from './Scale';
+import { POINTER, OBSERVER } from '../constants/roles';
+import { SIMPLE } from '../constants/scales';
 
 export class Room extends Component {
   constructor(props) {
     super(props);
     this.state = {
       pointers: [],
       observers: [],
-      room: '621QKO',
+      room: 'ABC123',
       showPoints: false,
       points: null,
       voted: false,
-      view: POINTERS,
+      view: POINTER, // TODO: default to player's own role
```

Comment:
Actually, never mind, it makes more sense for this to always default to POINTER view, no matter the role of the user